crates/voicevox_core/src/status.rs

```diff
@@ -168,37 +168,20 @@ fn open_libraries(root_dir_path: &Path) -> Result<BTreeMap<String, bool>> {
 unsafe impl Send for Status {}
 
 impl Status {
-    pub const METAS_STR: &'static str =
-        include_str!(concat!(env!("CARGO_WORKSPACE_DIR"), "/model/metas.json"));
-
     pub fn new(root_dir_path: &Path, use_gpu: bool, cpu_num_threads: u16) -> Self {
         Self {
             root_dir_path: root_dir_path.to_path_buf(),
             light_session_options: SessionOptions::new(cpu_num_threads, false),
             heavy_session_options: SessionOptions::new(cpu_num_threads, use_gpu),
-            supported_styles: BTreeSet::default(),
             libraries: None,
             usable_libraries: BTreeSet::new(),
             usable_model_data_map: BTreeMap::new(),
             usable_model_map: BTreeMap::new(),
             speaker_id_map: BTreeMap::new(),
-            metas_str: String::new(),
+            metas_str: CString::new("").unwrap(),
```

Comment:
こっちの方が`unwrap`が消えるのでよいのではないかと。

```suggestion
            metas_str: CString::default(),
```

Reply:
ありがとうございます！
CStringの空文字列の作り方がわからず、困っていたところでした...!